src/native/constants/navigation.js

```diff
@@ -13,14 +12,21 @@ export default {
   },
 
   tabProps: {
-    swipeEnabled: false,
-    activeBackgroundColor: 'rgba(255,255,255,0.1)',
-    inactiveBackgroundColor: Colors.brandPrimary,
-    tabBarStyle: { backgroundColor: Colors.brandPrimary },
+    swipeEnabled: true,
     tabBarPosition: 'bottom',
+    activeTintColor: 'black',
```

Comment:
On mine I think this appears yellow?

Reply:
yea that prop applies to the overall tint of the tab. the bottom line is called the Indicator, and styled with other props. but lets leave this for later.